Parent Entity Planned End Date unit/ParentEntityPlannedEndDateunit.js

```diff
@@ -95,8 +95,14 @@ tau.mashups
             _.extend(registry.units, registry.register(units));
         }
 
-        globalConfigurator.getGlobalBus().once('configurator.ready', function(e, configurator) {
-            addUnits(configurator);
+        var appConfigurator;
+        globalConfigurator.getGlobalBus().on('configurator.ready', function(e, configurator) {
+
+            if (!appConfigurator && configurator._id && configurator._id.match(/board/)) {
+
+                appConfigurator = configurator;
+                addUnits(appConfigurator);
+            }
```

Comment:
DRY principle violation. The same code is duplicated in all custom unit mashups. When something changes you will have to fix all the realizations and re-deploy all the mashups affected.